packages/config/index.js

```diff
@@ -40,14 +74,13 @@ function resolvePaths(config) {
         }
       })(config[key]);
     });
-  return config;
+  return result;
 }
 
 function normalizeURLs(config) {
   var basePath = config.basePath.replace(/^\/*/, '/').replace(/\/*$/, '');
   var assetPath = config.assetPath.replace(/(^\/*|\/*$)/g, '');
-
-  return Object.assign(config, {
+  return Object.assign({}, config, {
     locations: config.locations
       .map(function(location) {
         return basePath + location.replace(/\/*$/, '').replace(/^\/*/, '/');
```

Comment:
Same as above - we should allow optional trailing slashes.

Reply:
For the asset path, I fail to see a good reason for a change. I feel we only normalise a URL path that is only ever used internally to build asset URLs. Why should we change that mechanism?